test/functional/wallet_multiwallet.py

```diff
@@ -108,7 +107,7 @@ def wallet_file(name):
 
         # should not initialize if wallet path can't be created
         exp_stderr = "boost::filesystem::create_directory:"
-        self.nodes[0].assert_start_raises_init_error(['-wallet=wallet.dat/bad'], exp_stderr, match=ErrorMatch.PARTIAL_REGEX)
+        self.nodes[0].assert_start_raises_init_error(['-wallet=w8/bad'], exp_stderr, match=ErrorMatch.PARTIAL_REGEX)
```

Comment:
This change isn't strictly necessary, but using `w8` instead of `wallet.dat` here makes the test more robust so `default_wallet_filename` can be changed in the future. The top level wallet directory will always contain a `w8` file, but may or may not contain a `wallet.dat` file, depending on the `default_wallet_filename` value.